dockers/docker-fpm-frr/bgpd.conf.default.j2

```diff
@@ -98,10 +154,10 @@ router bgp {{ DEVICE_METADATA['localhost']['bgp_asn'] }}
     neighbor PEER_V6 soft-reconfiguration inbound
     neighbor PEER_V6 route-map TO_BGP_PEER_V6 out
   exit-address-family
-{% endblock peers_peer_group %}
-{% block bgp_peers_with_range %}
-{% if BGP_PEER_RANGE %}
-{% for bgp_peer in BGP_PEER_RANGE.values() %}
+{# endblock peers_peer_group #}
+{# block bgp_peers_with_range #}
```

Comment:
Can you please return all of your conversions of `{% %}` to `{# #}` back?

Reply:
@pavel-shirshov  I need these changes, or else to delete the lines, what do you think about?